### PR TITLE
Add some hints on how to find the pool IDs

### PR DESCRIPTION
**Pull request by ericzolf**
It's not obvious which ID is coming from where, hence extend the docs to explain it.

### CODE REVIEW

File: plugins/modules/activation_key.py [hunk 1]
```diff
@@ -62,12 +62,15 @@
...
           - Mutually exclusive with I(name) and I(upstream_pool_id).
+          - Also named I(Candlepin Id) in the CSV export of the subscriptions,
+          - it is as well the UUID as output by I(hammer subscription list).
         type: str
         required: false
       upstream_pool_id:
         description:
           - Upstream Pool ID of the Subscription to be added.
           - Mutually exclusive with I(name) and I(pool_id).
+          - Also named I(Master Pools) at access.redhat.com.
```
Comment (evgeni) [inline plugins/modules/activation_key.py]:
```suggestion
          - Also named C(Master Pools) in the Red Hat Portal.
```

`I()` is for option names, so I'd avoid that here. See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation

And I think we should call the portal by name, not url (and if url, then `U(https://access.redhat.com)`)

File: plugins/modules/activation_key.py [hunk 2]
```diff
@@ -62,12 +62,15 @@
         description:
           - Pool ID of the Subscription to be added.
           - Mutually exclusive with I(name) and I(upstream_pool_id).
+          - Also named I(Candlepin Id) in the CSV export of the subscriptions,
+          - it is as well the UUID as output by I(hammer subscription list).
```
Comment (evgeni) [inline plugins/modules/activation_key.py]:
```suggestion
          - Also named C(Candlepin Id) in the CSV export of the subscriptions,
          - it is as well the C(UUID) as output by C(hammer subscription list).
```

Is the "CSV" export still a thing in modern Satellite/Katellos?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: plugins/modules/activation_key.py (hunk 2 above)
Comment (ericzolf) [reply plugins/modules/activation_key.py]: And, yes, in Satellite 6.8 / Katello 3.16, CSV export is still a thing, even if I'm not sure it's modern :-)

image

**Comment by ericzolf** [comment]
All changes accepted, thanks @evgeni !

File: plugins/modules/activation_key.py (hunk 2 above)
Comment (evgeni) [reply plugins/modules/activation_key.py]: Oh, cool. I thought about `hammer csv` which is gone. :+1:

**Review by evgeni — APPROVED** [review]

**Pull request merged**